client/my-sites/site-settings/form-general.jsx

```diff
@@ -552,16 +553,14 @@ export class SiteSettingsFormGeneral extends Component {
 	}
 
 	renderAdminInterface() {
-		const { site, isSimple } = this.props;
-		if (
-			! isEnabled( 'layout/wpcom-admin-interface' ) &&
-			( ! isEnabled( 'layout/dotcom-nav-redesign-v2' ) ||
-				( isEnabled( 'layout/dotcom-nav-redesign-v2' ) && isSimple ) )
-		) {
-			return null;
+		const { site, siteSlug, isSimple } = this.props;
+		if ( ! isEnabled( 'layout/dotcom-nav-redesign-v2' ) || isSimple ) {
+			return isEnabled( 'layout/wpcom-admin-interface' ) ? (
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)

Sorry, I may be missing something, but do we need `! isEnabled( 'layout/dotcom-nav-redesign-v2' )`?
`'layout/dotcom-nav-redesign-v2'` is enabled except for Jetpack Cloud, which does not have General Settings. 

Would the following code work well?

```suggestion
		if ( isSimple && isEnabled( 'layout/wpcom-admin-interface' ) {
			return <SiteAdminInterfaceExperiment
```

Reply:
I'd prefer to keep the flags for a while, and we can remove them all at once later 🙂

Reply:
Make sense!